src/components/DataTable.js

```diff
@@ -0,0 +1,56 @@
+import * as React from 'react';
```

Comment:
Ensure that you both understand everything in this file before merging in changes! This will be necessary for building on this in the future.